src/app/dashboard/accounts/[guid]/page.tsx

```diff
@@ -38,29 +38,19 @@ export default function AccountPage({ params }: AccountPageProps): JSX.Element {
     );
   }
 
+  let infoComponent: JSX.Element;
+  if (isInvestment(account)) {
+    infoComponent = <InvestmentInfo account={account} />;
+  } else if (isAsset(account) || isLiability(account)) {
```

Comment:
AssetInfo also shown in Liability accounts. I'm not 100% all widgets are useful. For example liability accounts typically will not have income (unless there is some cashback/bonus things in a credit card). Leaving like this for now as other are potentially useful